docs/user/nifi/install.rst

```diff
@@ -1,3 +1,5 @@
+.. _nifi_install:
+
 Installation
```

Comment:
Can we update this to be "GeoMesa NiFi Installation" so that the ref to it reads as...

"Follow the steps under *GeoMesa NiFi Installation* to install the GeoMesa processors in your NiFi instance"

?

Basically, I'm looking for that sentence to have a bit more info/context about what one is installing.

Reply:
man, that's not even a line I added ;)

if we update it though, it also has the effect of updating the sidebar menu to be kind of redundant and verbose...

you don't think the rest of the sentence provides enough context?

Reply:
Yeah, it is fine.  I prefer verbose links / descriptions, but yeah, if it'll mess with the sidebar, we can leave it as well.